test/behaviour/concept/type/thingtype/ThingTypeSteps.java

```diff
@@ -322,13 +322,10 @@ public void thing_type_set_plays_role_throws_exception(RootLabel rootLabel, Stri
         assertThrows(() -> get_thing_type(rootLabel, typeLabel).asRemote(tx()).setPlays(roleType));
     }
 
-    @When("{root_label}\\( ?{type_label} ?) set plays role: {scoped_label} as {type_label}")
-    public void thing_type_set_plays_role_as(RootLabel rootLabel, String typeLabel, Parameters.ScopedLabel roleLabel, String overriddenLabel) {
+    @When("{root_label}\\( ?{type_label} ?) set plays role: {scoped_label} as {scoped_label}")
+    public void thing_type_set_plays_role_as(RootLabel rootLabel, String typeLabel, Parameters.ScopedLabel roleLabel, Parameters.ScopedLabel overriddenLabel) {
```

Comment:
It was inconsistent that we accepted unscoped role labels here, given that we require them in literally every other step. `behaviour` has been updated to reflect this in https://github.com/graknlabs/behaviour/pull/138